crates/secrets-types/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "wasmcloud-secrets-types"
```

Comment:
I didn't see a workflow for publishing this crate. Should we add that here or is it ok to wait until after this merges?

Reply:
I was planning on waiting until after this merges